webthing/client.py

```diff
@@ -46,6 +46,7 @@ def _process_observation(self, relative_uri: str, json_str: str) -> None:
             callback(relative_uri, observation)
 
     def _ws_subscribe(self, relative_uri: str, callback: Callable[[str, str], None]) -> str:
+        self._ws = StompWebsocket(self._ws_uri)
```

Comment:
Method _ws_subsribe is called multiple times during the lifetime of the webthing so initializing Websocket here is inappropriate. I guess this was to avoid error spamming when websocket not available (determined by retry timeout parameter on StompWebsocket).
Try to initialize websocket when calling _ws_subscribe for the first time and maybe implement consecutive retry fail condition?